src/universal/modules/notifications/components/TeamInvite/TeamInvite.js

```diff
@@ -52,17 +54,15 @@ const TeamInvite = (props) => {
         {'.'}
       </div>
       <div className={css(defaultStyles.button)}>
-        <Button
+        <StyledButton
           aria-label='Accept team invitation'
```

Comment:
I noticed you changed the sentence case to title case for auth, but you're leaving it here. Do you have a guideline on when to use title case vs. sentence case? 
(Personally I like sentence case everywhere. seems friendlier, more conversational).

Reply:
I like sentence case too, but in the case of buttons I feel like they feel more consistent when using title case for the visible label. Here the button label is simply “Accept” and the `aria-label` is sentence case. I tend to prefer sentence case for `aria-label` and `title` attributes. I’ve noticed Bojan tends to default to sentence case for headings, but Taya typically changes those to title case. So we’re kind of back and forth. I can start a doc to track (and reach some level of consent) and we can tighten up as we go.

Reply:
yeah, i think we should reach consensus & be explicit about this in our design guidelines (if we don't have one yet, we should probably make one, too. design around buttons & when to use what types would be good)

Reply:
For sure! See #2232